test/include/partitioner/PartitionerWeight.h

```diff
@@ -0,0 +1,38 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef PARTITIONERWEIGHT_H
+#define PARTITIONERWEIGHT_H
+
+// MOOSE includes
+#include "MooseEnum.h"
+#include "PetscMatPartitioner.h"
+
+class PartitionerWeight;
```

Comment:
Can you please name this `PartitionerWeightTest`... just so that it's clear that it's a "test" object?